aspnetcore/toc.yml

```diff
@@ -936,11 +936,13 @@
               uid: security/data-protection/compatibility/index
             - name: Replace machineKey in ASP.NET
               uid: security/data-protection/compatibility/replacing-machinekey
-    - name: Protect secrets in development
-      displayName: password
-      uid: security/app-secrets
-    - name: Azure Key Vault Configuration Provider
-      uid: security/key-vault-configuration
+    - name: Secrets management
+      items:
+        - name: Protect secrets in development
+          displayName: password
```

Comment:
consider adding "connection string", plus any other terms you can think of for the kinds of secrets being managed.  Then put the same displayName text on the Key Vault node.